docs/development/vsphere-dev.md

```diff
@@ -8,6 +8,53 @@ We are using [#sig-onprem channel](https://kubernetes.slack.com/messages/sig-onp
 ## Process
 Here is a [list of requirements and tasks](https://docs.google.com/document/d/10L7I98GuW7o7QuX_1QTouxC0t0aEO_68uHKNc7o4fXY/edit#heading=h.6wyer21z75n9 "Kops-vSphere specification") that we are working on. Once the basic infrastructure for vSphere is ready, we will move these tasks to issues.
 
+## CoreDNS
```

Comment:
Nit: 'Setting up DNS' or 'DNS setup' would be a more generic and intuitive heading.

And we can add few lines about why CoreDNS is required-  vSphere doesn't have any inbuilt DNS,  that's why we are providing support for external DNS.